### PR TITLE
Validations and model specs

### PR DESCRIPTION
**Pull request by YKerroum**
- Add the following validations:
  - For the User model:
    - Name must not be blank.
    - PostsCounter must be an integer greater than or equal to zero.
  - For the Post model:
    - Title must not be blank.
    - Title must not exceed 250 characters.
    - CommentsCounter must be an integer greater than or equal to zero.
    - LikesCounter must be an integer greater than or equal to zero.
- Add unit specs for all of your models' methods and validations.

### CODE REVIEW

File: app/models/post.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 class Post < ApplicationRecord
+  validates :title, presence: true, length: { maximum: 1000, too_long: '1000 characters is the maximum allowed' }
```
Comment (anagudelogu) [inline app/models/post.rb:2]: - [ ] You did a great job working on this milestone! 💯 However, according to the requirements, the title's length must not exceed 250 characters and not 1000. Please consider changing this! 🤓 🔧 

image

```suggestion
  validates :title, presence: true, length: { maximum: 250, too_long: '250 characters is the maximum allowed' }
```

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+require 'spec_helper'
+
+RSpec.describe User, type: :model do
+  context 'validations' do
+    it 'name should be present' do
+      user = User.new(name: nil)
+      expect(user.valid?).to eq(false)
```
Comment (anagudelogu) [inline spec/models/user_spec.rb:8]: - [ ] **[OPTIONAL]** You could write this spec in a better readable way using the `be_valid` helper:

```suggestion
      expect(user).not_to be_valid
```

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @YKerroum, 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:✔️ ✔️ 

Hi @YKerroum ✔️ ✔️ 

Your project is complete! Great job making the changes requested by the previous reviewer🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request✔️ 
- validations well implemented✔️ 
- unit test for models ✔️ 
- No linter errors✔️ 

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**